graylog2-server/src/test/java/org/graylog2/contentpacks/facades/InputFacadeTest.java

```diff
@@ -338,12 +369,89 @@ public void delete() throws NotFoundException {
         final Input input = inputService.find("5acc84f84b900a4ff290d9a7");
         final InputWithExtractors inputWithExtractors = InputWithExtractors.create(input);
 
-        assertThat(inputService.totalCount()).isEqualTo(2L);
+        assertThat(inputService.totalCount()).isEqualTo(3L);
         facade.delete(inputWithExtractors);
 
-        assertThat(inputService.totalCount()).isEqualTo(1L);
+        assertThat(inputService.totalCount()).isEqualTo(2L);
         assertThatThrownBy(() -> inputService.find("5acc84f84b900a4ff290d9a7"))
                 .isInstanceOf(NotFoundException.class);
     }
 
+    @Test
+    @UsingDataSet(locations = "/org/graylog2/contentpacks/inputs.json", loadStrategy = LoadStrategyEnum.CLEAN_INSERT)
+    public void resolveNativeEntity() throws NotFoundException {
+        when(lookupuptableBuilder.lookupTable("whois")).thenReturn(lookupuptableBuilder);
+        when(lookupuptableBuilder.build()).thenReturn(lookupTable);
+        when(lookupTableService.newBuilder()).thenReturn(lookupuptableBuilder);
+        when(lookupTableService.hasTable("whois")).thenReturn(true);
+        when(dbLookupTableService.get("whois")).thenReturn(Optional.of(lookupTableDto));
+        when(lookupTableDto.id()).thenReturn("dead-beef");
+
+        final Input input = inputService.find("5ae2eb0a3d27464477f0fd8b");
+        EntityDescriptor entityDescriptor = EntityDescriptor.create(ModelId.of(input.getId()), ModelTypes.INPUT_V1);
+        EntityDescriptor expectedEntitiyDescriptor = EntityDescriptor.create(ModelId.of("dead-beef"), ModelTypes.LOOKUP_TABLE_V1);
+        Graph graph = facade.resolveNativeEntity(entityDescriptor);
```

Comment:
The `.resolveNativeEntity()` method returns a `Graph<EntityDescriptor>` so we should not use the raw type here but the generic type:

```suggestion
        Graph<EntityDescriptor> graph = facade.resolveNativeEntity(entityDescriptor);
```